include/trace/events/sof.h

```diff
@@ -98,6 +98,23 @@ TRACE_EVENT(sof_stream_position_ipc_rx,
 	TP_printk("device_name=%s", __get_str(device_name))
 );
 
+TRACE_EVENT(sof_ipc4_fw_config,
```

Comment:
missing commit description. checkpatch would have complained about this

Reply:
yes, we need a commit message